pkg/api/types.go

```diff
@@ -181,10 +198,13 @@ type Binding struct {
 	Status BindingStatus
 }
 
+// BindingSpec represents a description of a Binding.
 type BindingSpec struct {
 	// InstanceRef is the reference to the Instance this binding is to.
+	// Immutable.
 	InstanceRef kapi.ObjectReference
-	// AppLabelSelector selects the pods in the Binding's namespace that should be injected with the results of the binding
+	// AppLabelSelector selects the pods in the Binding's namespace that
+	// should be injected with the results of the binding.  Immutable.
 	AppLabelSelector kapi.LabelSelector
```

Comment:
Why was a ref to a SIP removed?  (sorry about missing that conversation)
I thought that we were going to create a generic SIP object (for use beyond this SIG) and then we'd have a ref to it from here.  With AppLabelSelector, don't we run the risk of it being out of sync with the corresponding SIP? Also, don't we need a ref to the SIP so we know which to delete when we unbind?